server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java

```diff
@@ -51,6 +58,43 @@
                 .body(result);
     }
 
+    @RequestMapping("/rss/fr_FR")
+    @ApiOperation(value = "Articles RSS (fr)", httpMethod = "GET")
+    @JsonView(FlatView.class)
+    public String getArticlesRssFr() throws IOException, FeedException {
+        return getArticlesFeed(SessionLanguage.fr);
+    }
+
+    @RequestMapping("/rss/us_US")
+    @ApiOperation(value = "Articles RSS (en)", httpMethod = "GET")
+    @JsonView(FlatView.class)
+    public String getArticlesRssEn() throws IOException, FeedException {
+        return getArticlesFeed(SessionLanguage.en);
+    }
+
+    @JsonView(FlatView.class)
+    public String getArticlesFeed(SessionLanguage language) throws IOException, FeedException {
+        SyndFeed feed = new SyndFeedImpl();
```

Comment:
why do you need to annotate this internal method with ```java @JsonView(FlatView.class)```
Your method can be set with a protected visibility